### PR TITLE
[stable/sentry] fix: reverts changes to sentry secrets template

### PR DESCRIPTION
**Pull request by wilmardo**
#### What this PR does / why we need it:

Fixes regression created in #19938 

This if was leftover from an approach later abandoned in favor of the current. Sorry it got merged in, I looked over it when submitting :neckbeard: 

#### Which issue this PR fixes

https://github.com/helm/charts/pull/19938#discussion_r365110150

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[stable/mychartname]`)

### CODE REVIEW

**Comment by wilmardo** [comment]
@donbeave Could you verify if this fixes the regression?

**Review by donbeave — APPROVED** [review]

**Comment by donbeave** [comment]
@wilmardo LGTM

File: stable/sentry/templates/secrets.yaml [hunk 1]
```diff
@@ -22,9 +22,9 @@ data:
   {{ else }}
   user-password: {{ randAlphaNum 16 | b64enc | quote }}
   {{ end }}
-  {{ if and (.Values.postgresql.existingSecret) (or (not .Values.postgresql.enabled) (.Values.postgresql.password)) }}
```
Comment (lalinsky) [inline stable/sentry/templates/secrets.yaml:25]: The correct variable is `.Values.postgresql.postgresqlPassword`.

Comment (lalinsky) [reply stable/sentry/templates/secrets.yaml:25]: And I think there should be `and` here, not `or`, like in the case of redis below.

**Pull request merged**